src/js/constants/NameMappings.js

```diff
@@ -0,0 +1,13 @@
+var AppStatus = require("../constants/AppStatus");
+
+const NameMappings = {
+  statusNameMapping: {
```

Comment:
How about regrouping all of these view-bound constants into an ViewConstants file that exports the various objects?